src/agent/Core/ApplicationPool/Group/ProcessListManagement.cpp

```diff
@@ -61,14 +61,20 @@ Process *
 Group::findProcessWithStickySessionIdOrLowestBusyness(unsigned int id) const {
 	int leastBusyProcessIndex = -1;
 	int lowestBusyness = 0;
-	unsigned int i, size = enabledProcessBusynessLevels.size();
+	unsigned int i, size = enabledProcessBusynessLevels.size(), highest_gen = 0;
 	const int *enabledProcessBusynessLevels = &this->enabledProcessBusynessLevels[0];
 
 	for (i = 0; i < size; i++) {
 		Process *process = enabledProcesses[i].get();
+        highest_gen = max(highest_gen, process->generation);
```

Comment:
Should use `std::max()` to be unambiguous. I've seen platforms on which a plain `max()` fails to compile due to namespace ambiguity.